docker-entrypoint.sh

```diff
@@ -50,14 +78,64 @@ if [ "$1" = 'rabbitmq-server' ]; then
 		done
 		cat >> /etc/rabbitmq/rabbitmq.config <<-'EOF'
 			      {loopback_users, []}
+		EOF
+
+		if [ "$RABBITMQ_MANAGEMENT_ENABLED" ]; then
```

Comment:
I'm really not a big fan of having this extra environment variable hard-coded in the `:management` tag -- is there any command we could use to instead check whether the plugin is enabled directly instead of relying on anyone who's interested in the management plugin to be using the tag explicitly?

Something like this seems promising:
```bash
if [ "$(rabbitmq-plugins list -m -e rabbitmq_management)" ]; then
	# ...
```